packages/prisma-client/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@webstudio-is/prisma-client",
+  "version": "0.0.1",
+  "description": "Webstudio Prisma layer",
+  "author": "Webstudio <github@webstudio.is>",
+  "homepage": "https://webstudio.is",
+  "scripts": {
+    "checks": "tsc --noEmit",
+    "build:prisma": "prisma format && prisma generate",
+    "migrate": "dotenv -e ../../apps/designer/.env -- prisma migrate"
```

Comment:
we should probably figure out a way to avoid having designer here

Reply:
e.g. if we call this from designer, .env was already read, so when do we really call migrate in this package without first having the env?